### DISCUSSION TITLE
I want to join in Team

### DISCUSSION THREAD

**Discussion by mevadadarshan19** [ROM-builders/temporary#18723] [General]
My Trees :
Device : https://github.com/Mevadadarshan1997/android_device_asus_X01BD
Common : https://github.com/Mevadadarshan1997/android_device_asus_sdm66
Vendor : https://github.com/Mevadadarshan1997/proprietary_vendor_asus
Kernel : https://github.com/ArrowOS-Devices/android_kernel_asus_X01BD

Successful builds : 
https://cirrus-ci.com/task/6144294801637376
https://cirrus-ci.com/task/4833579365040128
https://cirrus-ci.com/task/6600414573887488
https://cirrus-ci.com/task/6312936960950272

There many more successful build i was done.

Thank you....

**Comment by Apon77** [comment]
please follow #learn in telegram group

**Reply by mevadadarshan19** [reply]
I already did all things

**Reply by Apon77** [reply]
all things means? what did you do?
